<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="free">
       <attribute name="label">
        <string>Free</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>731</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>condition:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_4">
           <property name="plainText">
            <string>{

}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnSend">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>170</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="windows">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>windows</string>
       </attribute>
       <widget class="QPushButton" name="btnGetWindows">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>get windows</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSource">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>pageSource</string>
       </attribute>
       <widget class="QPushButton" name="btnGetPageSource">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>get page source</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>tabId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTabId"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="newWindow">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>newWindow</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>parentWindowId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditParentWindowId"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnOpenNewWindow">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>open new window</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>url:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditUrl"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="enterValue">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>enterValue</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>tabId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTabId_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnEnterValue">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>enterValue</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>selector:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSelector"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditValue"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="clickElement">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>clickElement</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>tabId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTabId_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>selector:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSelector_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnClickElement">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>clickElement</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="evaluateScript">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>evaluateScript</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>tabId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTabId_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>411</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>script:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="plainText">
            <string>function main() {
alert('123');
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnEvaluateScript">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>0</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>evaluateScript</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="monitorFor">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="label">
        <string>monitorFor</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_13">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>411</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>script:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="plainText">
            <string>function main() {
alert('123');
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>301</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>condition:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="plainText">
            <string>function condition() {
return true;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnMonitorFor">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>180</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>monitorFor</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_26">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>monitorId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTabId_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
